Google.PowerShell/BigQuery/GcbqTable.cs

```diff
@@ -112,4 +112,174 @@ protected override void ProcessRecord()
             }
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Creates a new empty table in the specified project and dataset.
+    /// </para>
+    /// <para type="description">
+    /// Creates a new empty table in the specified dataset. A Table can be supplied by object 
+    /// via the pipeline or the -InputObject parameter, or it can be instantiated by value 
+    /// with the flags below. The Dataset ID can be specified by passing in a string to 
+    /// -DatasetId, or you can pass a Dataset or DatasetReference to the -Dataset parameter. 
+    /// If no Project is specified, the default project will be used. This cmdlet returns 
+    /// a Table object.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> New-GcbqTable “new_tab” -Dataset “my_data” -Description “Some nice data!” 
```

Comment:
Nit: Code blocks preserve whitespace.  Multi-line code examples should start on a new line so the intended alignment matches the actual alignment. 

This line should also have a back-tick ( \` ) character at the end to indicate it continues on the next line. Otherwise, someone who copies and pastes this example will get errors about `-Expiration` not being a valid operator.